src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -592,4 +623,14 @@ private ItemBuilder createSmallAngryPlanetItemBuilder() {
       .withVolume("smallAngryPlanetVolume")
       .withYearCaption(Collections.singletonList("2019"));
   }
+
+  private ItemBuilder createSmallAngryPlanetItemBuilder(UUID locationId, int barcode) {
+
+    return new ItemBuilder()
+      .withPermanentLoanType(materialTypesFixture.book().getId())
+      .withMaterialType(loanTypesFixture.canCirculate().getId())
+      .withBarcode(Integer.toString(barcode))
```

Comment:
Barcode is a string, so why do we pass it as an integer just to convert to a string at the end?

Reply:
done